src/humanize/time.py

```diff
@@ -98,30 +105,29 @@ def naturaldelta(value, months=True, minimum_unit="seconds"):
     tmp = Unit[minimum_unit.upper()]
     if tmp not in (Unit.SECONDS, Unit.MILLISECONDS, Unit.MICROSECONDS):
         raise ValueError(f"Minimum unit '{minimum_unit}' not supported")
-    minimum_unit = tmp
+    min_u = tmp
```

Comment:
Let's keep the original name here, it's more descriptive (or at least `min_unit`)

Reply:
The problem is, the `minimum_unit` variable is defined in the function signature as a `str`, and mypy does not approve of reassigning variables to different types. I didn't want to change the name of the argument, because that could break other people calling the functions using keywords.

So the renaming is a bit awkward, but I'm not sure how else to do it.

Reply:
Ah, I see.

Can we rename it to `min_unit` then?

Reply:
Sure.